src/components/ebay-carousel/style.less

```diff
@@ -1,115 +1,135 @@
-.carousel {
-    @transition-time: 0.25s;
+@ebay-carousel-transition-time: 0.25s;
+@ebay-carousel-transition-function: ease-out;
+@ebay-carousel-control-enabled-opacity: 0.92;
+@ebay-carousel-control-disabled-opacity: 0.3;
```

Comment:
Does it need the `ebay-` prefix? I doubt there would be collisions by only having `carousel-`.

Reply:
Just a thought. No request to change it.